workload-identity/README.md

```diff
@@ -125,3 +125,67 @@ periodics:
       - auth
       - list
 ```
+
+### Migrate Prow Job to Use Workload Identity
+
+> :Note: **Workload identity works best with pod utilities**: Migrate to use pod
+> utilities if there is no `decoration: true` on your job, and come back to this
+> doc once that's done.
+
+#### Background
+
+Prow jobs run in k8s pods, and each pod is responsible for uploading artifacts
+to designated remote storage location(e.g. GCS), this upload is done by a
+container called `sidecar`. To be able to upload to GCS, `sidecar` container
+will need to way to authenticate to GCP, which was historically done by GCP
+service account key(normally stored as `service-account` secret in build
+clusters), this key is mounted onto `sidecar` container by prow, which it uses
+for authenticating with GCP.
+
+Workload identity is a keyless solution from GCP that appears to be more secure
+than storing keys around, and prow also supports this by running the entire pod
```

Comment:
```suggestion
than storing keys, and Prow also supports this by running the entire pod
```

Reply:
`around` doesn't add any detail, `Prow` should probably be consistently capitalized as a proper noun.